spec/models/waste_carriers_engine/registration_spec.rb

```diff
@@ -604,7 +604,8 @@ module WasteCarriersEngine
             registration.metaData.activate!
 
             # Use .to_i to ignore milliseconds when comparing time
-            expect(registration.reload.expires_on.to_i).to eq(3.years.from_now.to_i)
+            expect(registration.reload.expires_on.to_i)
```

Comment:
This spec fails on this PR. There is a one second difference between the two dates. Can you spot where I might have introduced this?